### PR TITLE
zeplin: Add version 5.1.1

### PR DESCRIPTION
**Pull request by Bibz87-CG**
I had to create this manifest to use Zeplin via Scoop so I thought I should publish it so others could benefit, too. 😃

### CODE REVIEW

File: bucket/zeplin.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "bin": "Zeplin.exe",
+    "shortcuts": [
+        [
+            "Zeplin.exe",
+            "Zeplin"
+        ]
+    ],
+    "checkver": {
+        "url": "https://pkg.zeplin.io/windows/latestv2/x64/RELEASES",
+        "regex": "Zeplin-(\\d+.\\d+.\\d+)-full",
```
Comment (Ash258) [inline bucket/zeplin.json:26]:
```suggestion
        "regex": "Zeplin-([\\d.]+)-full",
```

File: bucket/zeplin.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://pkg.zeplin.io/windows/latestv2/x64/Zeplin-$version-full.nupkg#/dl.7z"
+            },
+            "32bit": {
+                "url": "https://pkg.zeplin.io/windows/latestv2/ia32/Zeplin-$version-full.nupkg#/dl.7z"
+            }
+        }
```
Comment (Ash258) [inline bucket/zeplin.json:37]: Add hash extraction from `$baseurl/RELEASES`

File: bucket/zeplin.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+    "homepage": "https://zeplin.io/",
+    "description": "Connected space for product teams. Handoff designs and styleguides with accurate specs, assets, code snippets—automatically.",
+    "license": "MIT",
+    "version": "5.1.1",
+    "architecture": {
+        "64bit": {
+            "url": "https://pkg.zeplin.io/windows/latestv2/x64/Zeplin-5.1.1-full.nupkg#/dl.7z",
```
Comment (Ash258) [inline bucket/zeplin.json:8]: rename to 7zip is not needed

File: bucket/zeplin.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
+{
+    "homepage": "https://zeplin.io/",
+    "description": "Connected space for product teams. Handoff designs and styleguides with accurate specs, assets, code snippets—automatically.",
+    "license": "MIT",
```
Comment (Ash258) [inline bucket/zeplin.json:4]: Is this just a client for product or you need to have valid license to use the software?

image

File: bucket/zeplin.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
+{
+    "homepage": "https://zeplin.io/",
+    "description": "Connected space for product teams. Handoff designs and styleguides with accurate specs, assets, code snippets—automatically.",
```
Comment (Ash258) [inline bucket/zeplin.json:3]: Use something like: `Collaborative tool for handoff designs with specs, ...`

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/zeplin.json (hunk 2 above)
Comment (Bibz87-CG) [reply bucket/zeplin.json:37]: Is there a way to reverse reverse the regex, like there is on `checkver`? I haven't found a way to filter `$baseurl/RELEASES` to only return the latest version and the latest version happens to be at the bottom of the list.

Comment (Ash258) [reply bucket/zeplin.json:37]: `$sha1\\s+$basename`

File: bucket/zeplin.json (hunk 4 above)
Comment (Bibz87-CG) [reply bucket/zeplin.json:4]: I use it via my client's company work environment so I can't say for certain, but it looks like Zeplin only requires an account to work; I logged in with my Google account and it let me in straight away without any payment information.

File: bucket/zeplin.json (hunk 5 above)
Comment (Bibz87-CG) [reply bucket/zeplin.json]: Fixed!

File: bucket/zeplin.json (hunk 3 above)
Comment (Bibz87-CG) [reply bucket/zeplin.json]: Fixed!

File: bucket/zeplin.json (hunk 2 above)
Comment (Bibz87-CG) [reply bucket/zeplin.json:45]: Fixed!

File: bucket/zeplin.json [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+        "reverse": true
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://pkg.zeplin.io/windows/latestv2/x64/Zeplin-$version-full.nupkg",
+                "hash": {
+                    "url": "https://pkg.zeplin.io/windows/latestv2/x64/RELEASES",
+                    "regex": "$sha1\\s+$basename"
+                }
```
Comment (Ash258) [inline bucket/zeplin.json:36]: There should be no problem with generic hash

```suggestion
                "url": "https://pkg.zeplin.io/windows/latestv2/x64/Zeplin-$version-full.nupkg"
```

File: bucket/zeplin.json [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+            },
+            "32bit": {
+                "url": "https://pkg.zeplin.io/windows/latestv2/ia32/Zeplin-$version-full.nupkg",
+                "hash": {
+                    "url": "https://pkg.zeplin.io/windows/latestv2/ia32/RELEASES",
+                    "regex": "$sha1\\s+$basename"
+                }
```
Comment (Ash258) [inline bucket/zeplin.json:43]:
```suggestion
                "url": "https://pkg.zeplin.io/windows/latestv2/ia32/Zeplin-$version-full.nupkg"
```

File: bucket/zeplin.json [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+            }
+        }
```
Comment (Ash258) [inline bucket/zeplin.json:45]:
```suggestion
        },
        "hash": {
            "url": "$baseurl/RELEASES",
            "regex": "$sha1\\s+$basename"
        }
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/zeplin.json (hunk 4 above)
Comment (Bibz87-CG) [reply bucket/zeplin.json:4]: Good enough to add to scoop extras?

Comment (Ash258) [reply bucket/zeplin.json:4]: It is definitely not MIT and it is definitely not free. The limitation for 1 project and account requirement is too strict.

File: bucket/zeplin.json [hunk 9]
```diff
@@ -0,0 +1,43 @@
+{
+    "homepage": "https://zeplin.io/",
+    "description": "Collaborative tool for designs' and styleguides' handoff with specs, assets and code snippets.",
+    "license": "MIT",
+    "version": "5.1.1",
```
Comment (Ash258) [inline bucket/zeplin.json:5]:
```suggestion
    "version": "5.1.1",
    "description": "Collaborative tool for designs' and styleguides' handoff with specs, assets and code snippets",
    "homepage": "https://zeplin.io",
    "license": {
        "identifier": "Proprietary",
        "url": "https://zeplin.io/terms"
    },
```

File: bucket/zeplin.json [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+            "url": "https://pkg.zeplin.io/windows/latestv2/x64/Zeplin-5.1.1-full.nupkg",
+            "hash": "sha1:ada573adf43ce547c815706a1356d739227fef77"
+        },
+        "32bit": {
+            "url": "https://pkg.zeplin.io/windows/latestv2/ia32/Zeplin-5.1.1-full.nupkg",
+            "hash": "sha1:17a032825fa4d3914c948e47f8377f7dd55451b2"
+        }
+    },
+    "extract_dir": "lib\\net45",
+    "bin": "Zeplin.exe",
```
Comment (Ash258) [inline bucket/zeplin.json:17]: What are available parameters?

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/zeplin.json (hunk 10 above)
Comment (Bibz87-CG) [reply bucket/zeplin.json:20]: I've never used Zeplin via command line but `zeplin -h` returns this: 
```
Usage: Zeplin.exe [options]

Options:
  -V, --version             output the version number
  -e, --export <path>       export using dump at path
  -f, --squirrel-firstrun   squirrel-firstrun
  -i, --squirrel-install    squirrel-install
  -u, --squirrel-updated    squirrel-updated
  -r, --squirrel-uninstall  squirrel-uninstall
  -o, --squirrel-obsolete   squirrel-obsolete
  -h, --help                output usage information```

`--export` doesn't work on my computer, `-V` wrongly returns 1.0.0 and `squirrel` options are related to Zeplin's auto-updater.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**